Algorithmia/client.py

```diff
@@ -114,6 +114,11 @@ def invite_to_org(self,orgname,username):
         response = self.putHelper(url,data={})
         return response
 
+    def get_build_logs(self,user,algo):
+        url = '/v1/algorithms/'+user+'/'+algo+'/builds'
```

Comment:
This works; however ideally we would use the Algorithmia API Client for this. In the short term I'd be happy with merging this in as it will function as expected; but if we change the underlying API route this would break.